source/includes/write/replace.py

```diff
@@ -0,0 +1,31 @@
+# start-replace-one
+restaurants = database.get_collection("restaurants")
```

Comment:
We typically use the form `database["restaurants"]`

Reply:
Good to know! Fixed